app/commands/create_claim_command.rb

```diff
@@ -0,0 +1,17 @@
+class CreateClaimCommand < SerialSequenceCommand
```

Comment:
Ill explain this - I have created a more specific command rather than the generic serial sequence command.
This has allowed me to do all the specific stuff in this new command as well as fire the correct event which used to be done in the controller